### PR TITLE
Ecs task definitions missing methods

### PR DESCRIPTION
**Pull request by soumyo13**
### Description

Docs Changes and Added the missing methods.

### Issues Resolved

Docs Changes and Added the missing methods.

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [x] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/aws_stepfunctions_state_machine.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+| name | The name of the state machine. |
+| status | The current status of the state machine. |
+| definition | The Amazon States Language definition of the state machine. |
+| role_arn | The ARN of the IAM role used when creating this state machine. |
+| type | The type of the state machine. Valid values: `STANDARD` or `EXPRESS`. |
+| creation_date | The creation date of the state machine. |
+| logging_configuration (level) | The category of execution history in which events are logged. |
+| logging_configuration (include_execution_data) | Whether execution data is included in your log. Valid values: `true` or `false`. |
+| logging_configuration (destinations) | An array of objects that describes where your execution history events will be logged. |
+| logging_configuration (enabled) | Selects whether or not the state machine's AWS X-Ray tracing is enabled. Valid values: `true` or `false`. |
```
Comment (IanMadd) [inline docs/resources/aws_stepfunctions_state_machine.md:42]: Should `logging_configuration (enabled)` be `tracing_configuration (enabled)` ?

**Review by IanMadd — APPROVED** [review]
I left a question on aws_stepfunctions_state_machine.md, otherwise this looks good.

File: docs/resources/aws_stepfunctions_state_machine.md (hunk 1 above)
Comment (soumyo13) [reply docs/resources/aws_stepfunctions_state_machine.md:42]: Yeah it should be tracing_configuration. I corrected it.

**Comment by soumyo13** [comment]
> I left a question on aws_stepfunctions_state_machine.md, otherwise this looks good.

Thanks Ian.

**Review by ppradhan9 — APPROVED** [review]
Thanks! @soumyo13

**Review by ppradhan9 — APPROVED** [review]

**Review by Nirbhay1997 — APPROVED** [review]

**Pull request merged**